imagepicker/src/main/AndroidManifest.xml

```diff
@@ -15,7 +15,7 @@
         <activity
             android:name="com.esafirm.imagepicker.features.ImagePickerActivity"
             android:configChanges="orientation|screenSize"
-            android:label="@string/app_name"
```

Comment:
I think we should remove it completely from the codebase since it will not take any effect to the library. CMIIW